app/views/layouts/_google_analytics.html.erb

```diff
@@ -4,7 +4,7 @@
   <script>
     window.dataLayer = [{
       'signedIn': '<%= user_signed_in?.to_json %>',
-      'hasIPs': '<%= (!!current_user && current_organisation.ips.any?).to_json %>'
+      'hasIPs': '<%= (!!current_user && current_organisation && current_organisation.ips.any?).to_json %>'
```

Comment:
Rather than `current_organisation && current_organisation.ips.any?` can we use `current_organisation&.ips&.any?`

Same applies anywhere we're using `current_organisation && current_organisation.ips.any?`